azure-iot-device/azure/iot/device/iothub/pipeline/pipeline_stages_iothub.py

```diff
@@ -42,7 +42,7 @@ def _run_op(self, op):
             # sees this -1, it will send a GetTwinOperation to refresh desired properties.
 
             if op.feature_name == constant.TWIN_PATCHES:
-                logger.info(
+                logger.debug(
```

Comment:
i feel this should be info as it contains information about enabling a certain feature

Reply:
This process of enabling is hidden to the user though due to being implicit though. They aren't really exposed to the concept of features being enabled/disabled

---
In reply to: [477604104](https://github.com/Azure/azure-iot-sdk-python/pull/649#discussion_r477604104) [](ancestors = 477604104)